components/hubspot/sources/new-contact/new-contact.js

```diff
@@ -0,0 +1,59 @@
+const hubspot = require("../../hubspot.app.js");
+
+module.exports = {
+  key: "hubspot-new-contact",
+  name: "New Contacts",
+  description: "Emits an event for each new contact added.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    hubspot,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+
+  async run(event) {
+    const lastRun = this.db.get("createdAfter") || this.hubspot.monthAgo();
+    const createdAfter = new Date(lastRun);
+    const data = {
+      limit: 10,
+      sorts: [
+        {
+          propertyName: "createdate",
```

Comment:
Similar question - can this be filtered by created date or another time-based param to reduce the result set?

Reply:
For this API call, createdate is not one of the searchable properties, but it does allow you to sort by that property.